src/components/SplToken/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { useTokenList } from 'src/hooks/tokenList'
+
+export const SplToken = (props: { mint: string }) => {
+  const tokenMap = useTokenList()
+
+  const token = tokenMap.get(props.mint)
```

Comment:
I recommend using the hook in the parent component and passing the token to this component.